apps/client/app/employee/[id]/manager-report/[cycleId]/page.tsx

```diff
@@ -13,7 +13,6 @@ import { apolloClient } from '@/lib/apollo-client'
 import { queries } from '@/lib/graphql-queries'
```

Comment:
The page will only show peer reviews if they have a `reviewerId`. This is necessary because all the peer reviews are empty by default now.